packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -160,27 +175,27 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           id={ id }
           tabIndex={ disabled ? -1 : 0 }
           aria-expanded={ isOpen ? 'true' : 'false' }
-          role='combobox'
```

Comment:
This element would actually need to stay a combobox -- it should be a combo whether it's single or multiselect, the only difference is we should add `aria-multiselectable="true/false"`.

The issue with having a single-select be a listbox is that listboxes do not have an expand/collapse functionality, and would set up a screen reader user's expectations that it behave like a listbox. The issue with not having a role at all is then we have a focusable generic element that still has a name/description and provides no information on how to interact with it.

Reply:
@smhigley Thanks for your comments. I have a question here: If I use role='combobox', but Accessibility Insights will scan a bug: **Required ARIA child role not present: textbox.** 
And this error also occurs in https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html and https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown

![image](https://user-images.githubusercontent.com/60056296/118444653-02dff400-b720-11eb-8744-fc72a2ffeafd.png)


![image](https://user-images.githubusercontent.com/60056296/118444599-ed6aca00-b71f-11eb-8dd8-a58568d47916.png)

Reply:
Ah, I should've mentioned that off the bat 😅. Including the `textbox` role as a child of a combobox is the ARIA 1.1 pattern, and actually has significantly worse screen reader support in practice than leaving it off (and if included, it would need to be the element that gets focus). Having the `combobox` be the single interactive element is the ARIA 1.2 pattern, which has good screen reader support across the board.

Unfortunately, axe-core has yet to update its rules for combobox, so it still flags it as an error 😭. The issue open against them is here: https://github.com/dequelabs/axe-core/issues/2505

For now, it'd still be better to go with the pattern with good support (no textbox), and ignore/skip the automated error until they fix it.

Reply:
Thank you @smhigley for letting me know about this. It is really a good chance for me to learn more about ARIA pattern and axe-core. Really appreciate your support.